### DISCUSSION TITLE
Add --no-open flag to cli dev

### DISCUSSION THREAD

**Discussion by Dueen** [remix-run/remix#3311] [Proposals]
Add the option to run remix watch (and/or dev) with a --no-open flag to prevent cli from automatically open the app in a tab

Reason: My default web browser is not always set to the one where I want to open the project at. Some times I just want to start the server and see if it's working then later come back and open the app in my browser of choice.

**Comment by kiliman** [comment]
Interesting. My browser doesn’t automatically open when I start the app. I typically Cmd+click on the URL in the terminal to launch. What terminal/IDE are you using?

**Reply by sergiodxa** [reply]
Same, this is not a feature of the Remix CLI

**Reply by Dueen** [reply]
Indeed this seems to be a Cloudflare wrangler feature
